src/components/enhanced-table/EnhancedTable.jsx

```diff
@@ -33,13 +33,20 @@ const EnhancedTable = ({ fetchService, externalFilter }) => {
   const { loading, fetchData } = useAxios({ service: fetchService, fetchOnMount: false })
 
   const getData = useCallback(async () => {
+    let status = null
```

Comment:
can this will work?
const status  = externalFilter.status !== 'all' && [externalFilter.status]

Reply:
It can. I fixed it.